aten/src/ATen/native/ForeachUtils.h

```diff
@@ -134,7 +134,18 @@ bool check_fast_path_restrictions(
         return false;
       }
       if (tensorLists[0][j].strides() != tensor_list[j].strides()) {
-        return false;
```

Comment:
Should probably verify that the length of tensor_list[j].strides is the same as tensorLists[0][j].strides()

Reply:
addressed in b50604ae7ad86746350d16a8dc2d0f683d71f6cf